### ISSUE TITLE
[1.16.5] Small conflict with Requiem mod

### ISSUE THREAD

**Issue by KamiSami** [kwpugh/gobber_fabric#17]
Hi, there is a problem with this mod installed along with Requiem mod, causing monsters you possess, not all of them though (i think), but skeletons and spiders and creepers at least seems to have something weird going on them, they normally shouldn't aggro you when possessing monster, but with this mod, for example the skeletons aggro you and drop aggro and repeats that behavior, sometimes manages shoot arrow too. This might be something fixed on your side possibly? I made issue to Requiem but it may seem you can do something about this rather. https://github.com/Ladysnake/Requiem/issues/350

**Comment by kwpugh** [comment]
Is this happening in one of the Gobber dimensions or in the Overworld?

**Comment by KamiSami** [comment]
Overworld at least, haven't tested in gobber dimension.

**Comment by kwpugh** [comment]
I looked through my code.  Gobber does nothing with any of the mob behavior.  The only changes to mobs are with their equipment when spawned AND that only kicks off if the mob is spawned in one of the Gobber dimensions.

I also looked at some of the Requiem source code, in particular the mixing that affect the mob classes that Gobber mixing into, we are not hitting the same methods, so I don't see where a conflict could occur.
